youtube_dl/extractor/indavideo.py

```diff
@@ -60,8 +60,12 @@ def _real_extract(self, url):
 
         formats = [{
             'url': video_url,
-            'height': self._search_regex(r'\.(\d{3,4})\.mp4$', video_url, 'height', default=None),
+            'height': int_or_none(self._search_regex(r'\.(\d{3,4})\.mp4$', video_url, 'height', default=None)),
         } for video_url in video_urls]
+        
+        # Filter out redundant null height format
+        formats = [fmt for fmt in formats if fmt['height']]
```

Comment:
Oh, I didn't check carefully enough, the URL does have the height as it's the same apart from the query string, but the regex match is returning None since it's anchored to the end.